### PR TITLE
Added dir of all necessary items for graphing FB RSVP data

### PR DESCRIPTION
**Pull request by mattjefferson27**

### CODE REVIEW

**Comment by samertm** [comment]
Looks good to me! :ship:

File: dashboard/chart/chart.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+<div class="ct-chart ct-octave" style="width: 950px;"></div>
+
+<script>
+//this is the location of the csv file
+//the format of it needs to be like this:
+//		monthyear,num_attendees
+//		2014-01,16
+//		2014-02,37
+//		2014-03,38
+var CSV_FILE = 'monthly_attendees.csv';
```
Comment (mjperrone) [inline dashboard/chart/chart.html]: In order to get this automatically updating, you'll have to make an ajax request to http://dxetech.org/dashboard/monthly_attendees.csv

**Comment by mjperrone** [comment]
I think this actually be in another repository. This isn't actually going to be running on the server, it's just going to be copy-pasted into the SquareSpace UI, so I don't think it should be in this one. I don't think any other repository we have necessarily makes sense, so maybe we need a new repository for SS code?

What do y'all think?

**Comment by samertm** [comment]
That makes sense to me. @mattjefferson27, feel free to make a "squarespace" repo and move this code there.

**Comment by mjperrone** [comment]
Moved it to private/squarespace.

**Comment by mjperrone** [comment]
And pointed it to `dxetech.org/dashboard/montly_attendees.csv`

**Comment by mjperrone** [comment]
and put it on squarespace! yay!
http://directactioneverywhere.com/meta

**Comment by samertm** [comment]
YAY

**Pull request merged**